packages/react-aria-components/src/DropZone.tsx

```diff
@@ -80,19 +80,20 @@ function DropZone(props: DropZoneProps, ref: ForwardedRef<HTMLDivElement>) {
         [FileTriggerContext, {ref: fileTriggerRef}],
         [TextContext, {id: textId, slot: 'heading'}]
       ]}>
+      {/* eslint-disable-next-line */}
```

Comment:
what was this for?

Reply:
yeah so this was because i added an onClick to dropzone so that when you clicked on it, it would focus on the button allowing you to paste since button has the ClipboardProps. the linting errors had to do with accessibility, wanting to add some sort of key press handler to the dropzone (you can read the errors [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/f0d2ddb65f21278ad29be43fb167a1092287b4b1/docs/rules/no-static-element-interactions.md) and [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/f0d2ddb65f21278ad29be43fb167a1092287b4b1/docs/rules/click-events-have-key-events.md). but since the dropzone has a hidden button, there is keyboard functionality to paste so i felt like it was a special case in which these could be ignored. if not, i think it's fine we remove onClick to dropzone and have it so you must keyboard paste.